src/NServiceBus.RavenDB/Internal/RavenDBVersionValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus.Persistence.RavenDB;
+
+using System;
+using NuGet.Versioning;
+
+static class MinimumRequiredRavenDbVersion
```

Comment:
Looks like we forgot to change the file name, and I suggest we change this to MinimumRequiredRavenDbServerVersion